src/nat/front_ends/fastapi/auth_flow_handlers/websocket_flow_handler.py

```
@@ -129,7 +133,7 @@ async def _handle_oauth2_auth_code_flow(self, config: OAuth2AuthCodeFlowProvider
         await self._web_socket_message_handler.create_websocket_message(_HumanPromptOAuthConsent(text=authorization_url)
                                                                         )
         try:
-            token = await asyncio.wait_for(flow_state.future, timeout=300)
+            token = await asyncio.wait_for(flow_state.future, timeout=self._auth_timeout_seconds)
         except TimeoutError as exc:
             raise RuntimeError("Authentication flow timed out after 5 minutes.") from exc
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)